icons/forgejo/forgejo-plain-wordmark.svg

```diff
@@ -0,0 +1 @@
4 2.358 3.309 3.537 5.264 3.537s3.698-1.179 5.227-3.537c1.554-2.428 2.77-5.758 3.647-9.99.877-4.232 1.316-9.054 1.316-14.465 0-5.48-.439-10.302-1.316-14.465-.877-4.231-2.093-7.527-3.647-9.885-1.53-2.428-3.272-3.643-5.227-3.643zm85.433 0c-1.98 0-3.735 1.215-5.264 3.643-1.53 2.358-2.745 5.654-3.647 9.885-.878 4.163-1.316 8.984-1.316 14.465 0 5.411.438 10.233 1.316 14.465.902 4.232 2.118 7.562 3.647 9.99 1.554 2.358 3.309 3.537 5.264 3.537s3.698-1.179 5.227-3.537c1.554-2.428 2.77-5.758 3.647-9.99.877-4.232 1.316-9.054 1.316-14.465 0-5.48-.439-10.302-1.316-14.465-.877-4.231-2.093-7.527-3.647-9.885-1.53-2.428-3.272-3.643-5.227-3.643zm-28.91.121c-1.905 0-3.61 1.214-5.113 3.642-1.48 2.359-2.645 5.655-3.497 9.887-.853 4.231-1.279 9.053-1.279 14.464s.439 10.232 1.316 14.464c.903 4.232 2.143 7.562 3.723 9.99 1.604 2.359 3.447 3.538 5.527 3.538 1.63 0 3.108-.763 4.437-2.289 1.328-1.526 2.419-3.746 3.271-6.66l-1.73-5.619c-.726 2.359-1.591 4.127-2.594 5.307-1.003 1.11-2.106 1.665-3.309 1.665-1.504 0-2.832-.833-3.986-2.498-1.128-1.734-2.017-4.162-2.669-7.284a45.141 45.141 0 0 1-.495-3.255l16.062-8.712c-.05-5.342-.514-9.99-1.391-13.945-.853-4.023-1.993-7.145-3.422-9.365-1.429-2.22-3.046-3.33-4.85-3.33Zm0 7.492c1.304 0 2.457.833 3.46 2.498 1.002 1.596 1.78 3.78 2.33 6.556.353 1.694.584 3.534.721 5.487L82.09 81.098c-.012-.558-.032-1.101-.032-1.678 0-4.024.288-7.528.864-10.511.577-2.983 1.38-5.307 2.407-6.972 1.053-1.665 2.244-2.498 3.572-2.498zm-56.523.088c1.354 0 2.557.832 3.61 2.497s1.88 4.023 2.482 7.076c.626 3.053.94 6.626.94 10.719 0 4.024-.314 7.597-.94 10.719-.602 3.052-1.43 5.445-2.482 7.18-1.053 1.664-2.256 2.498-3.61 2.498s-2.557-.834-3.61-2.499c-1.052-1.734-1.892-4.127-2.519-7.18-.601-3.121-.902-6.694-.902-10.718 0-4.093.3-7.666.902-10.72.627-3.052 1.467-5.41 2.52-7.075 1.052-1.665 2.255-2.497 3.61-2.497zm33.827 0c1.404 0 2.645.798 3.723 2.393 1.077 1.527 1.917 3.712 2.519 6.556.626 2.844.94 6.14.94 9.887 0 3.746-.314 7.076-.94 9.99-.602 2.844-1.441 5.064-2.52 6.66-1.077 1.595-2.318 2.392-3.722 2.392-1.379 0-2.607-.797-3.685-2.393-1.078-1.596-1.93-3.815-2.557-6.66-.602-2.913-.902-6.243-.902-9.99 0-3.746.3-7.042.902-9.886.627-2.844 1.48-5.03 2.557-6.556 1.078-1.595 2.306-2.393 3.685-2.393zm51.606 0c1.354 0 2.557.832 3.61 2.497s1.88 4.023 2.481 7.076c.627 3.053.94 6.626.94 10.719 0 4.024-.313 7.597-.94 10.719-.601 3.052-1.429 5.445-2.481 7.18-1.053 1.664-2.256 2.498-3.61 2.498s-2.557-.834-3.61-2.499c-1.053-1.734-1.892-4.127-2.519-7.18-.601-3.121-.902-6.694-.902-10.718 0-4.093.3-7.666.902-10.72.627-3.052 1.466-5.41 2.52-7.075 1.052-1.665 2.255-2.497 3.609-2.497zm-16.222 2.001v49.093c0 3.33-.276 5.828-.827 7.493-.527 1.665-1.291 2.497-2.294 2.497-1.228 0-2.23-.866-3.008-2.6l-1.053 6.555c.501 1.18 1.128 2.046 1.88 2.601.752.555 1.542.833 2.369.833 1.955 0 3.46-1.56 4.512-4.682 1.053-3.122 1.58-7.354 1.58-12.697V61.528ZM4.479 91.82c1.023 0 1.825 2.217 1.825 5.05 0 2.833-.802 5.05-1.825 5.05-1.024 0-1.825-2.217-1.825-5.05 0-2.833.801-5.05 1.825-5.05z"/></svg>
```

Comment:
Aspect ratio is wrong, logo is squished

so what did i need to change here:

- take the forgejo-original-wordmark.svg and merge all paths into a single one
- change color to black
- resized the image to 128x128 using iloveimg
- optimize the image using vecta

```suggestion
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 128 128"><path d="M19.73 40.855c-1.645 0-3.09.902-3.867 2.238h-1.852a11.81 11.81 0 0 0-11.773 11.77v17.105C.906 72.75 0 74.195 0 75.84a4.5 4.5 0 0 0 4.477 4.477 4.5 4.5 0 0 0 4.477-4.477c0-1.645-.902-3.09-2.238-3.871v-4.711a7.26 7.26 0 0 1 7.297-7.293h1.852a4.49 4.49 0 0 0 3.867 2.238 4.5 4.5 0 0 0 4.477-4.477 4.5 4.5 0 0 0-4.477-4.477c-1.645 0-3.09.902-3.867 2.238h-1.852a11.71 11.71 0 0 0-7.297 2.551v-3.176a7.26 7.26 0 0 1 7.297-7.293h1.852a4.49 4.49 0 0 0 3.867 2.238 4.5 4.5 0 0 0 4.477-4.477 4.5 4.5 0 0 0-4.477-4.477zm0 2.652c1.023 0 1.824.801 1.824 1.824s-.801 1.824-1.824 1.824-1.824-.801-1.824-1.824.801-1.824 1.824-1.824zm83.207 10.211a4.03 4.03 0 0 0-4.004 4.008c0 2.195 1.809 4.004 4.004 4.004s4.004-1.809 4.004-4.004-1.805-4.008-4.004-4.008zM19.73 55.902c1.023 0 1.824.801 1.824 1.824s-.801 1.82-1.824 1.82-1.824-.801-1.824-1.82.801-1.824 1.824-1.824zm83.207.105a1.7 1.7 0 1 1 0 3.434c-.961 0-1.715-.754-1.715-1.715s.754-1.719 1.715-1.719zM32.359 59.57c-1.977 0-3.73.438-5.262 1.316a9.6 9.6 0 0 0-3.645 3.57c-.875 1.504-1.316 3.246-1.316 5.223 0 1.957.441 3.695 1.316 5.227.902 1.527 2.117 2.73 3.645 3.605 1.555.852 3.309 1.277 5.262 1.277s3.695-.426 5.227-1.277c1.551-.875 2.766-2.078 3.645-3.609s1.316-3.266 1.316-5.223c0-1.977-.441-3.719-1.316-5.223-.879-1.527-2.094-2.719-3.645-3.57-1.531-.879-3.27-1.316-5.227-1.316zm23.191 0c-1.93 0-3.547.391-4.848 1.164a6.17 6.17 0 0 0-2.48 2.602v-3.578h-3.008v19.844h3.156v-9.996c0-2.254.578-3.984 1.73-5.187 1.152-1.227 2.719-1.84 4.695-1.84.125 0 .25.012.379.035.125 0 .25.016.375.039zm10.652 0c-1.906 0-3.621.402-5.148 1.203-1.504.801-2.695 1.93-3.574 3.383-.875 1.43-1.312 3.094-1.312 5s.438 3.582 1.313 5.035a9.36 9.36 0 0 0 3.574 3.418c1.527.805 3.242 1.203 5.148 1.203 1.727 0 3.281-.363 4.66-1.09.84-.437 1.578-1 2.215-1.691l-.098 1.355c-.172 2.348-.578 4.094-1.73 5.223-1.152 1.152-2.945 1.727-5.375 1.727a12.99 12.99 0 0 1-4.281-.711c-1.355-.477-2.496-1.129-3.422-1.957l-1.578 2.445c1.078.953 2.457 1.676 4.133 2.18 1.703.523 3.457.789 5.262.789 3.332 0 5.852-.816 7.555-2.445 1.703-1.602 2.555-4.133 2.555-7.59 0-4.766.625-10.625-1.227-13.191-.828-1.43-1.941-2.492-3.348-3.195-1.375-.727-3.594-1.09-5.32-1.09zm51.547 0c-1.98 0-3.734.438-5.262 1.316a9.6 9.6 0 0 0-3.645 3.57c-.879 1.504-1.316 3.246-1.316 5.223 0 1.957.438 3.695 1.316 5.227.902 1.527 2.117 2.73 3.645 3.605 1.555.852 3.309 1.277 5.262 1.277s3.695-.426 5.223-1.277c1.555-.875 2.77-2.078 3.648-3.609s1.313-3.266 1.313-5.223c0-1.977-.437-3.719-1.312-5.223-.879-1.527-2.094-2.719-3.648-3.57-1.527-.879-3.27-1.316-5.223-1.316zm-28.895.043c-1.906 0-3.609.441-5.109 1.316-1.48.852-2.645 2.043-3.496 3.57s-1.277 3.27-1.277 5.223.438 3.695 1.313 5.227c.902 1.527 2.145 2.73 3.723 3.605 1.602.855 3.445 1.277 5.523 1.277 1.629 0 3.105-.273 4.434-.824s2.418-1.352 3.27-2.406l-1.727-2.027c-.727.852-1.59 1.488-2.594 1.914-1.004.402-2.105.602-3.309.602-1.5 0-2.828-.301-3.98-.902a6.91 6.91 0 0 1-2.672-2.629c-.199-.371-.367-.762-.504-1.176l16.066-3.148c-.051-1.93-.516-3.605-1.391-5.035-.852-1.453-1.992-2.582-3.422-3.383s-3.043-1.203-4.848-1.203zm0 2.707c1.301 0 2.457.301 3.457.902 1.004.578 1.777 1.363 2.332 2.367a6.39 6.39 0 0 1 .73 1.98l-13.34 2.574a10.36 10.36 0 0 1-.02-.609c0-1.453.289-2.719.863-3.793s1.379-1.918 2.406-2.52c1.051-.602 2.242-.902 3.57-.902zm-56.496.031c1.355 0 2.559.301 3.609.902s1.879 1.453 2.48 2.555c.625 1.105.938 2.395.938 3.871 0 1.453-.312 2.746-.937 3.871-.602 1.102-1.43 1.969-2.48 2.594-1.051.602-2.254.902-3.609.902s-2.555-.301-3.605-.902c-1.055-.625-1.891-1.492-2.52-2.594-.602-1.125-.902-2.418-.902-3.871 0-1.477.301-2.766.902-3.871.629-1.102 1.465-1.953 2.52-2.555s2.254-.902 3.605-.902zm33.813 0c1.402 0 2.641.289 3.719.863 1.078.555 1.918 1.34 2.52 2.367.625 1.027.938 2.219.938 3.574s-.312 2.555-.937 3.605c-.602 1.027-1.441 1.828-2.52 2.406s-2.316.863-3.719.863c-1.379 0-2.605-.289-3.684-.863s-1.93-1.379-2.555-2.406c-.602-1.051-.902-2.254-.902-3.605s.301-2.547.902-3.574c.625-1.027 1.477-1.812 2.555-2.367 1.078-.574 2.305-.863 3.684-.863zm51.578 0c1.352 0 2.555.301 3.609.902s1.879 1.453 2.48 2.555c.625 1.105.938 2.395.938 3.871 0 1.453-.312 2.746-.937 3.871-.602 1.102-1.43 1.969-2.48 2.594-1.055.602-2.258.902-3.609.902s-2.555-.301-3.609-.902c-1.051-.625-1.891-1.492-2.516-2.594-.602-1.125-.902-2.418-.902-3.871 0-1.477.301-2.766.902-3.871.625-1.102 1.465-1.953 2.516-2.555s2.258-.902 3.609-.902zm-16.215.723v17.73c0 1.203-.273 2.105-.824 2.707-.527.602-1.293.902-2.293.902-1.23 0-2.23-.312-3.008-.941l-1.051 2.367c.5.426 1.125.738 1.879.941s1.539.301 2.367.301c1.953 0 3.457-.566 4.508-1.691s1.578-2.656 1.578-4.586v-17.73zM4.477 74.016c1.023 0 1.824.801 1.824 1.824s-.801 1.824-1.824 1.824-1.824-.801-1.824-1.824.801-1.824 1.824-1.824zm0 0" fill="#000000"/></svg>

```